assets/js/googlesitekit/datastore/site/info.js

```diff
@@ -36,9 +36,16 @@ import { normalizeURL, untrailingslashit } from '../../../util';
 
 const { createRegistrySelector } = Data;
 
-function getSiteInfoProperty( propName ) {
+function getSiteInfoProperty( propName, castToBool = false ) {
 	return createRegistrySelector( ( select ) => () => {
 		const siteInfo = select( CORE_SITE ).getSiteInfo() || {};
+
+		// Added to convert keyMetricsSetupCompleted user ID to
+		// boolean, for flexibility it is made as parameter.
+		if ( castToBool && undefined !== siteInfo[ propName ] ) {
+			return !! siteInfo[ propName ];
+		}
```

Comment:
This isn't necessary since we can do the same in the one selector as this isn't a common need.